builder/vmware/common/driver_player5.go

```diff
@@ -35,7 +35,7 @@ func NewPlayer5Driver(config *SSHConfig) Driver {
 }
 
 func (d *Player5Driver) Clone(dst, src string, linked bool, snapshot string) error {
-	return errors.New("Cloning is not supported with VMWare Player version 5. Please use VMWare Player version 6, or greater.")
+	return errors.New("linked clones are not supported on this version")
```

Comment:
Same as with Fusion, I would hint that Player v5 is the issue and upgrading will fix the problem

Reply:
Simplified. Resolved in 67054cd.